src/sections/dataset/dataset-files/files-table/file-actions/file-actions-cell/file-action-buttons/file-options-menu/FileAlreadyDeletedModal.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button, Modal } from '@iqss/dataverse-design-system'
+import { ExclamationCircleFill } from 'react-bootstrap-icons'
+import styles from './FileAlreadyDeletedModal.module.scss'
+
+interface FileAlreadyDeletedModalProps {
+  show: boolean
+  handleClose: () => void
+}
+export function FileAlreadyDeletedModal({ show, handleClose }: FileAlreadyDeletedModalProps) {
+  return (
+    <Modal show={show} onHide={handleClose} size="lg">
+      <Modal.Header>
+        <Modal.Title>Edit File</Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        <p className={styles.paragraph}>
+          <ExclamationCircleFill />
+          This file has already been deleted (or replaced) in the current version. It may not be
```

Comment:
This should be moved to locales, for i18n.

Reply:
I forgot! Thanks! I added a commit, can you check it again?

Reply:
Looks good. Thanks!